src/mainwindow.cpp

```diff
@@ -83,7 +84,7 @@ MainWindow::MainWindow( const QString& filename, QString tmpFileName, bool fakeR
     mainLayout->addLayout( buttonLayout );
 
     setLayout( mainLayout );
-    setWindowTitle( "One Click Install" );
+    setWindowTitle( i18n("One Click Install") );
```

Comment:
Even I'm not sure.  It's up to you though. Few names cannot be translated for eg - YaST but OCI is made up of legit words.